server/conf.go

```diff
@@ -35,8 +32,6 @@ type Configuration struct {
 	SchemesUpdateInterval int `json:"schemes_update" mapstructure:"schemes_update"`
 	// Path to issuer private keys to parse
 	IssuerPrivateKeysPath string `json:"privkeys" mapstructure:"privkeys"`
-	// Issuer private keys
-	IssuerPrivateKeys map[irma.IssuerIdentifier]map[uint]*gabi.PrivateKey `json:"-"`
```

Comment:
I have removed this map here because I wanted to remove all private key caches, and because it would make it too easy for users to pass in cached private keys.

There is an alternative: passing your custom keyring to the `AddPrivateKeyRing()` method of the `irma.Configuration` struct (the `irma.Configuration` field on the `server.Configuration` struct is public).

Reply:
ah missed that way of injecting a custom keyring. In that case this is fine.